plugins/main/windows/overlay/README.md

```diff
@@ -0,0 +1,27 @@
+# L2Bridge  plugin (Windows)
+
+## Overview
+
+With Overlay plugin, all containers (on the same host) are plugged into an Overlay network based on VXLAN encapsulation. 
+
+
+## Example configuration
```

Comment:
why is this example configuration so different from the example in plugins/main/windows/overlay/sample.conf ?